src/components/Drawer/Drawer.stories.tsx

```diff
@@ -133,7 +133,7 @@ export const Sizes = {
             onClose={() => {
               setIsOpen1(false);
             }}
-            size={33}
+            size={'33%'}
```

Comment:
Optional: shall we change this to clamp to guide the users towards that?

Reply:
added, check latest commit